cdq-analysis/pkg/detect.go

```diff
@@ -122,21 +122,22 @@ func search(log logr.Logger, a Alizer, localpath string, devfileRegistryURL stri
 							}
 						}
 					}
-				} else if f.Name() == DockerfileName {
-					// Check for Dockerfile
+				} else if f.Name() == DockerfileName || f.Name() == AlternateDockerfileName {
```

Comment:
Small suggestion: Rather than checking for alternate types of Dockerfile names, what if we convert the name to lower case and check against "dockerfile"?

Reply:
then any other mix-case names are also allowed? i.e. `DoERerFiLE`?

Reply:
I think we still need to check the exact allowed name, due to https://github.com/redhat-appstudio/application-service/blob/902d8977998183611795a8d5eae5ead8aee04e95/pkg/devfile/devfile.go#L750-L756

When looking for the files using endpoint, we still need to check for supported names. 

But in the cdq analysis here, when we have the repo downloaded on disk, I can convert to lower case and check against that.

Reply:
updated